### PR TITLE
Update node version to latest LTS Fermium

### PR DESCRIPTION
**Pull request by gunjam**
This updates the node version in the `.nvmrc` file to the latest LTS release of 14.17.0 in order to resolve https://github.com/alphagov/govuk-frontend/issues/2162.

Node 14 is the active LTS version, with 16 being the 'current' version. 12 is now in maintenance.

Thanks.

### CODE REVIEW

**Comment by 36degrees** [comment]
Hi,
  
Thanks for raising. I'm going to approve this, but only on the basis that we’ve been meaning to move on to Node 14 for a while.
  
We really don’t want to encourage installation 'from source' by pointing directly at the GitHub repo.

The supported methods for installing GOV.UK Frontend are to:

- install the govuk-frontend package from npm
- use the pre-compiled files included with every release

I understand that you're doing this because it allows you to deliver an optimised bundle for users. It's great to see this focus on performance, and we're keen to make this easier as part of #711.

However, the contents of the GitHub repo, including the `src` folder, are considered outside of the 'contract' we have with our users. We may make changes to the repo structure or our release processes, for example, but these would not be considered breaking changes as long as the published package and pre-compiled files are not affected. We're also unlikely to make further changes to enable this unsupported installation method.

If you are decide to install from source anyway, please ensure you're pinning to a tag or commit hash that references a specific release, rather than pointing at the main branch. This is going to become especially important in the coming months as we start work on 4.0, during which time the `src` folder will contain unreleased breaking changes.
  
This'll also need a review from a second person from the team before we merge it.

**Review by 36degrees — APPROVED** [review]

**Comment by 36degrees** [comment]
Note for the rest of the team – this would be an 'internal' change which doesn't need to be scheduled as part of a release or documented in release notes.

**Review by lfdebrux — APPROVED** [review]

**Pull request merged**
